lib/brain.js

```diff
@@ -1109,9 +1109,9 @@ Brain.prototype.mapCryptoUnitsDisplay = function mapCryptoUnitsDisplay (coins) {
     const coinSettings = coinUtils.getCryptoCurrency(coin.cryptoCode)
     const defaultUnit = _.head(_.keys(coinSettings.units))
     const unitSelected = _.get('cryptoUnits')(coin)
-    const { displayScale, displayCode } = unitSelected
-      ? _.get(`units.${unitSelected}`)(coinSettings)
-      : _.get(`units.${defaultUnit}`)(coinSettings)
+    const unit = _.includes(unitSelected, _.keys(coinSettings.units)) ? unitSelected : defaultUnit
+
+    const { displayScale, displayCode } = _.get(`units.${unit}`)(coinSettings)
```

Comment:
Lodash's `_.get()` accepts an array for paths too: `_.get(['units', unit], coinSettings)`.